docs/src/rust-feature-support/intrinsics.md

```diff
@@ -270,6 +270,6 @@ Name | Support | Notes |
 `simd_rem`  | No | |
 `simd_shl`  | No | |
 `simd_shr`  | No | |
-`simd_shuffle*`  | No | |
+`simd_shuffle*`  | Yes | Doesn't check indexes at compile time ([#1960](https://github.com/model-checking/kani/issues/1960))|
```

Comment:
This gets checked during the verification, right?

Reply:
Yes, it shows as an "index out-of-bounds" failure.

Reply:
So I would propose that we don't bother adding this issue here.

Reply:
That makes sense, just updated #1960 to make this clear. Removed the issue while resolving merge conflicts.